### DISCUSSION TITLE
How to add a lovelace card with live stream ?

### DISCUSSION THREAD

**Discussion by Frazou1** [tsightler/ring-mqtt#592] [Support Q&A]
I configure ring mqtt and look fine. Now im trying to get a live stream but i dont know how ! someone can help me please ?

**Comment by tsightler** [comment]
Have you attempted to follow the [documentation](https://github.com/tsightler/ring-mqtt/wiki/Video-Streaming#home-assistant-generic-camera-configuration)?  If so, perhaps detail what part you are having issues with.

**Reply by Frazou1** [reply]
Yes i still have timeout for the stream.  
Error from stream worker: Error opening stream (INVALIDDATA, Invalid data found when processing input) rtsp://192.168.2.65:1883
Error from stream worker: Error opening stream (ERRORTYPE_111, Connection refused) rtsp://192.168.2.65
Error from stream worker: Error opening stream (ERRORTYPE_5, I/O error) rtsp://03cabcc9-ring-mqtt:8554/343ea428074a_live

**Reply by tsightler** [reply]
OK, so that's not about adding a lovelace card.  Only one of those URLs seems anywhere close to correct (the last one), so you'll need to look at the logs from ring-mqtt and see what it shows when you attempt to connect.  Were you able to add the generic camera as an entity?

**Reply by Frazou1** [reply]
My second url is definitly wrong :  https://homeassistant:8123{{ states.camera.front_camera_snapshot.attributes.entity_picture }}
 But not sure where i can configure.  Also for the rtsp url i should use the ip of home assistant right ?

**Reply by Frazou1** [reply]
I found that : 
![image](https://user-images.githubusercontent.com/89617565/227969579-7b93c7d7-b5f5-463e-8703-7a8b6e645d06.png)

**Reply by tsightler** [reply]
As the text says, you only need this to allow RTSP client outside of Home Assistant.
